script/Arcueid/Day 2/02_00_ARC02_4_2.txt

```diff
@@ -166,12 +168,14 @@ The inside of the room is quite different from what I had expected.
 [sha:4286f35f65d363e1253d0cb710ff03f1e7e4d20d]{
 -- Page 254, Offset 3280.
 -- 部屋の半分は座敷になっており、奥にはびっしりと箪笥が並んでいる。そこだけ完全に和風の造りだ。
-Half of the room is occupied by tatami mats, with a bunch of tansu chests closely packed in the back. That's the only part of the room that is completely Japanese.
+Half of the room is occupied by tatami mats, with a bunch of tansu chests closely packed in the back.
+ That half of the room is entirely Japanese in style.
 }
 [sha:dd26800398de6827d6b9a255bb23c372bac402cc]{
 -- Page 254, Offset 3281.
 -- 一方、もう半分は洋風ではあるものの質素な造りで、洋館の一室というより学校の保健室を思わせる。
-On the other hand, the other half of the room while still simple, is westernized, reminiscent of a school infirmary rather than a room in a Western-style building.
+In contrast, the other half of the room is more Western in appearance.
```

Comment:
I like the old version of this line tbh. Though should be edited with an extra comma after "half of the room,"